changelog.d/5546.bugfix

```diff
@@ -0,0 +1 @@
+If you reply to a redaction, the redacted text still appears
```

Comment:
This does not match the content of this PR, this is about message edition. can you update please?